src/components/Home/Home.tsx

```diff
@@ -51,15 +53,11 @@ const Home = ():JSX.Element => {
     .then(cleanData => setDetails(cleanData))
     .catch(error => setError(error))
   }
-
-  useEffect(() => {
```

Comment:
Awesome! So we don't need useEffect if we are passing the getMarkets as an event handler? Good to know.